### ISSUE TITLE
Expanded Operating Status Designs

### ISSUE THREAD

**Issue by Agile6MSkinner** [department-of-veterans-affairs/va.gov-cms#19804]
## Description
We will be adding new operating status options for facilities. No changes other than deciding on colors for places  where the new statuses display.

### User story
**AS A**
**I WANT**
**SO THAT**

### Scope / Impact analysis
_What, if anything, could break as a result of this change?_
_Engineer should assess this when approaching PR._

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
- [ ] **Functional testing:** 
- [ ] **Code review:** 
- [ ] **Design review:** 
- [ ] **Accessibility review:** 
- [ ] **Product acceptance:** 

## Acceptance criteria
_ACs should capture expected behavior, to inform test plans. Consider devices, documentation updates including KBs, change management, and [content model](https://prod.cms.va.gov/admin/structure/cm_document) when applicable._

- [ ] 
- [ ] Impact analysis is filled out and QA test plan is updated accordingly
- [ ] Does this product have an existing regression test plan?
	- [ ] Yes, link: 
	- [ ] No, regression test plan will be created as part of the work / points in this ticket

**Comment by davidmpickett** [comment]
Closing as duplicate of #18225
